io.github.rocket.editor.yml

```diff
@@ -0,0 +1,32 @@
+app-id: io.github.rocket.editor
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: editor
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=network
+  - --device=dri
+modules:
+  - name: Rocket
+    buildsystem: qmake
+    subdir: editor
+    post-install:
+      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps ../io.github.rocket.editor.svg
+      - install -Dm644 -t /app/share/applications/ ../io.github.rocket.editor.desktop
+      - install -Dm644 -t /app/share/metainfo ../io.github.rocket.editor.appdata.xml
+    sources:
+      - type: git
+        url: https://github.com/rocket/rocket.git
+        tag: v0.11.0
+      - type: patch
+        path: 0001-install-editor.patch
+        use-git: true
+      - type: file
+        path: io.github.rocket.editor.svg
```

Comment:
you should be able to just use the file in `contrib/logo.svg` instead.

Reply:
Yeah, that's the same file, good point.

Reply:
OK, done.